src/main/java/frc/robot/components/Limelight.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.components;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class Limelight{
+    public NetworkTable limeLightTable;
+    public void init(){
+        limeLightTable = NetworkTableInstance.getDefault().getTable("limelight");
+    }
+    public double getTx(){
+      NetworkTableEntry targetX = limeLightTable.getEntry("tx");
+      double tx = targetX.getDouble(0);
+      return tx;
+    }
+    public double geTa(){
+      NetworkTableEntry targetA = limeLightTable.getEntry("ta");
+      double ta = targetA.getDouble(0);
+        return ta;
+    }
+    public double getTy(){
+        NetworkTableEntry targetY = limeLightTable.getEntry("ty");
+        double ty = targetY.getDouble(0);
+        return ty;
+    }
+    public double getDistance(){
+      return getTy();// this will need to change IMMEDIATLY 
+    }
+}
```

Comment:
Let's hold off on merging the Limelight code for now. We'll need to start out by running the limelight code in a separate robot prject since our bench test setup has no other subsystems.